src/System.Composition.Hosting/src/Resources/Strings.resx

```diff
@@ -1,5 +1,64 @@
 <?xml version="1.0" encoding="utf-8"?>
 <root>
+  <!-- 
```

Comment:
please remove that comment. this is auto-generated by VS but we don't need it :)